src/components/layout/TransactionWorkflow/TransactionWorkflow.tsx

```diff
@@ -264,6 +271,7 @@ function TransactionWorkflow({
       switch (contractType) {
         case CONTRACT_TYPES.ANT: {
           switch (interactionType) {
+            case ANT_INTERACTION_TYPES.SET_TARGET_ID:
             case ANT_INTERACTION_TYPES.SET_TICKER:
             case ANT_INTERACTION_TYPES.SET_NAME: {
               return {
```

Comment:
still don't see the point of switch here

Reply:
type protection for accessing TRANSACTION_DATA_KEYS

Reply:
> type protection for accessing TRANSACTION_DATA_KEYS
not sure i follow, can you explain what you mean?

Reply:
If we dont use a switch, or two `if` statements checking the type in the interaction, where we access the TRANSACTION_DATA_KEYS